From d3e33fa89e10d1f5bc47f559afc2a130eace1943 Mon Sep 17 00:00:00 2001
From: Shimrra Shai <shimrrashai@gmail.com>
Date: Mon, 16 Dec 2024 15:41:51 -0600
Subject: [PATCH 060/335] UPSTREAM: dt-bindings: arm: rockchip: Add Firefly
 ITX-3588J board

Document board compatible bindings.

Signed-off-by: Shimrra Shai <shimrrashai@gmail.com>
Acked-by: Krzysztof Kozlowski <krzysztof.kozlowski@linaro.org>
Link: https://lore.kernel.org/r/20241216214152.58387-2-shimrrashai@gmail.com
Signed-off-by: Heiko Stuebner <heiko@sntech.de>

(cherry picked from commit 8886252102bd774656d19423b7d85e1ddd78f9c0)
Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 Documentation/devicetree/bindings/arm/rockchip.yaml | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/Documentation/devicetree/bindings/arm/rockchip.yaml b/Documentation/devicetree/bindings/arm/rockchip.yaml
index de94f28bedce..4a9106aa8530 100644
--- a/Documentation/devicetree/bindings/arm/rockchip.yaml
+++ b/Documentation/devicetree/bindings/arm/rockchip.yaml
@@ -167,6 +167,13 @@ properties:
           - const: engicam,px30-core
           - const: rockchip,px30
 
+      - description: Firefly Core-3588J-based boards
+        items:
+          - enum:
+              - firefly,itx-3588j
+          - const: firefly,core-3588j
+          - const: rockchip,rk3588
+
       - description: Firefly Core-PX30-JD4 on MB-JD4-PX30 baseboard
         items:
           - const: firefly,px30-jd4-core-mb
-- 
2.48.1

